src/Models/User.php

```diff
@@ -114,7 +114,7 @@ class User extends SentryUser implements HasPresenter
      */
     public function revisions()
     {
-        return $this->hasMany('GrahamCampbell\Credentials\Models\Revision');
+        return $this->hasMany(\Config::get('credentials.revision'));
```

Comment:
You need to actually import the full class. You can't assume the class alias exists. I see that mistake made so often.